cmd/serve/serve.go

```diff
@@ -43,49 +40,65 @@ func NewServeCmd() *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			logger.L.Debug("Initializing shutdown channel")
-			c, _ := signal.NotifyContext(cmd.Context(), syscall.SIGINT, syscall.SIGTERM)
+			ctx := context.Background()
```

Comment:
ideally we should use the context for the command `cmd.Context()`

Reply:
I didn't understand why you use the context returned there instead of creating your own? What are the advantages?

Reply:
the cmd's Context returns underlying command context if command is executed with `ExecuteContext` or a context created with `context.Background()` if no context is set so effectively in this case, the behavior would be the same. 

the reason to use the one from the command is that i.e. for test purpose you may want to set an custom context that has i.e. a deadline or that has some key associated. By not using it then we would lose such information.

Reply:
solved with https://github.com/sco1237896/sco-backend/pull/15/commits/f1d1b19430d57c43c073a48e5d090d562b7ff628